### ISSUE TITLE
explorer.exe is extremely slow after debloating

### ISSUE THREAD

**Issue by Jozkah** [Raphire/Win11Debloat#102]
**Describe the bug**
my explorer crashes and takes a long time to open after debloating

**Expected behavior**
for it not to lag?

is there any way to reverse it?

**Comment by Raphire** [comment]
Heya, 

Have you restarted your PC since running the script?

**Comment by Jozkah** [comment]
> Heya,
> 
> Have you restarted your PC since running the script?

this has been an issue for weeks

**Comment by Raphire** [comment]
I see. Could you please answer the following to help narrow down the issue:
* What exactly is slow? Is it only when opening a new explorer window?
* What happens when explorer crashes? Does the window just close out of nowhere or does it show a 'Not responding' error window?
* Did this issue start right after running the script?
* Did you run the script with the default settings or did you select your own. If you selected your own, what settings did you select for the script?
* Are you running Windows 10 or 11?
* Do you have any network drives connected to your pc?

**Comment by Jozkah** [comment]
What exactly is slow? Is it only when opening a new explorer window?

opening a new explorer window takes about 20 seconds to load



What happens when explorer crashes? Does the window just close out of nowhere or does it show a 'Not responding' error window?

no buttons work then it just stays there and eventually does not respond



Did this issue start right after running the script?

only after restarting the pc after running the script



Did you run the script with the default settings or did you select your own. If you selected your own, what settings did you select for the script?

default settings and re-enabled dark mode



Are you running Windows 10 or 11?

11



Do you have any network drives connected to your pc?

at this very moment no as i’ve been trying to diagnose the issue, but previously yes but they were all local to my second pc




Extra: i’ve resigned and decided to reset the pc in the meantime

**Comment by Raphire** [comment]
Thanks for getting back so quickly. 

I'm unsure what part of the script's defaults could cause this, as not many of the default changes affect Explorer directly. If I had to hazard a guess I'd say it's related to disabling bing web search, as that does affect Explorer's search too.

Please let me know if the issue is resolved after resetting your PC.

**Comment by Raphire** [comment]
@Jozkah Do you have any updates on this issue?

**Comment by Jozkah** [comment]
> @Jozkah Do you have any updates on this issue?

well it's now fixed after reinstalling windows so there's not much to add unfortunately, i still have the same network drives and the same settings, just didn't debloat windows

**Comment by Raphire** [comment]
Thanks for the update. I'm unfortunately unable to pinpoint what may've caused your issue. For that reason I'll leave this issue open for now in case anybody else runs into this issue aswell.

**Comment by Ianisop** [comment]
been running into the same problem for a while now, thought it was just a windows 11 bug, but now that you mention it after reinstalling and not debloating, it seems to work fine

**Comment by Kenionatus** [comment]
The issue does indeed seem to lie with Bing Web Search. I went through the steps described in https://github.com/Raphire/Win11Debloat/discussions/114 
> Bing Web Search in Windows Search can be restored by running the Enable_Bing_Cortana_In_Search.reg file that is provided by the script, You can find it by navigating to Win11Debloat > Regfiles > Undo.

to reenable it and the immense slowdown went away instantly.

**Comment by Raphire** [comment]
This issue seems to have gone inactive. I do know Windows 11 has seem some performance improvements to Windows Explorer, which has been notoriously slow with the move to WinUI elements. 

If anyone continues to have this issue, please create a new issue and refer this one.
